tools/test/travis-ci/scancode_evaluate_test.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+# Copyright (c) 2020 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+import importlib
+import os
+import sys
+from unittest import TestCase
+
+# TODO: fix scancode to match python naming conventROOTi
+SCANCODE_EVALUATE = importlib.import_module("scancode-evaluate")
+license_check = SCANCODE_EVALUATE.license_check
+
+ROOT = os.path.abspath(
+	os.path.join(os.path.dirname(__file__))
+)
+
+# path to stub files
+stub_path = ROOT + "/scancode_test/"
+
+# template copyright notices
+invalid_header_1 = "/* Copyright (C) Arm Limited, Inc - All Rights Reserved\
+ * Unauthorized copying of this. file, via any medium is strictly prohibited\
+ * Proprietary and confidential\
+ */"
+
+invalid_header_2 = "/* mbed Microcontroller Library\
+ * Copyright (c) 2006-2013 ARM Limited\
+ *\
+ * Licensed under the Apache License, Version 2.0 (the \"License\");\
+ * you may not use this file except in compliance with the License.\
+ * You may obtain a copy of the License at\
+ *\
+ *     http://www.apache.org/licenses/LICENSE-2.0\
+ *\
+ * Unless required by applicable law or agreed to in writing, software\
+ * distributed under the License is distributed on an \"AS IS\" BASIS,\
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\
+ * See the License for the specific language governing permissions and\
+ * limitations under the License.\
+ */"
+
+
+# implement test class
+class TestScancodeEvaluate(TestCase):
+	""" Test scancode evaluation script """
+
+	def test_scancode_case_1(self):
+		""" Test Case 1 -- faulty json file
+			@inputs scancode_test_1.json
+			@outputs -1 if any error in file licenses found
+		"""
+		expected_result = -1
+		test_json = ROOT + "/scancode_test/scancode_test_1.json"
+
+		# pass json path to test function
+		result = license_check(ROOT, test_json)
+
+		self.assertEqual(expected_result, result)
```

Comment:
```suggestion
		self.assertEqual(
		    -1,
		    license_check(ROOT, ROOT + "/scancode_test/scancode_test_1.json"
		 )
```

Similar changes in `test_scancode_case_2()`.